R/tableofdifferences.R

```diff
@@ -99,13 +99,25 @@
 #'     row labels specifying rows to remove from the returned table.
 #' @param column.names.to.remove Character vector or delimited string of
 #'     column labels specifying columns to remove from the returned table.
+#' @param means.test The type of test used to compare a table of means 
+#'  (i.e. from numeric variable). Usually taken from QSettings. 
+#'  One of "tTest", "zTest", "Nonparametric".  
+#' @param proportions.test The type of test used to compare a table of proportions 
+#'  (i.e. from PickOne or PickAny variable). Usually taken from QSettings. 
+#'  One of "tTest", "zTest", "Nonparametric".
+#' @param bessel.constant Either a 0 or 1 which is a correction factor used in some tests.
+#'  Usually taken from QSettings$StatisticalAssumptions.
+#' @param design.effect.constant Another value taken from QSettings$StatisticalAssumptions
```

Comment:
`design.effect.constant` can be any positive number? Can you mention that in the documentation and add a bit of detail about what it actually is instead of just "Another value"?

Reply:
I've added some more description and a link to the wiki in 72ff76b